src/nvim/eval.c

```diff
@@ -12221,11 +12247,11 @@ static void find_some_match(typval_T *argvars, typval_T *rettv, int type)
       goto theend;
     }
     if (l != NULL) {
-      li = tv_list_find(l, start);
+      idx = tv_list_uidx(l, start);
+      li = tv_list_find(l, idx);
```

Comment:
Why precompute `idx` if it is recomputed in this function?

Reply:
Ah, `idx` could be used later on. If this is the case computation of `idx` could be moved after the `if` block.